cmake/internal/GridToolsConfig.cmake.in

```diff
@@ -26,6 +26,7 @@ get_filename_component(GRIDTOOLS_CONFIG_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" P
 if(NOT @PROJECT_NAME@_BINARY_DIR AND NOT TARGET GridTools::gridtools)
     include(CMakeFindDependencyMacro)
     find_dependency(Boost @REQUIRED_BOOST_VERSION@)
+    find_dependency(HPX QUIET)
```

Comment:
This should eventually require a minimum version. The `HPX::*` CMake targets are only on master at the moment and will be part of the next release. I recommend we require that version once it's out since I guess noone has a big need to support older versions of HPX.